x/blobstream/module.go

```diff
@@ -79,14 +77,16 @@ func (AppModuleBasic) RegisterGRPCGatewayRoutes(clientCtx client.Context, mux *r
 	}
 }
 
-// GetTxCmd returns the capability module's root tx command.
+// GetTxCmd returns no command because the blobstream module was disabled in app
+// version 2.
 func (a AppModuleBasic) GetTxCmd() *cobra.Command {
-	return bscmd.GetTxCmd()
+	return nil
 }
 
-// GetQueryCmd returns the capability module's root query command.
+// GetQueryCmd returns no command because the blobstream module was disabled in
+// app version 2.
 func (AppModuleBasic) GetQueryCmd() *cobra.Command {
-	return bscmd.GetQueryCmd()
+	return nil
```

Comment:
same as above

Reply:
This is disabling the CLI command but the intention is that this PR will only be merged to `main` because it's breaking. It won't be backported to v1.x. So v1.x binaries will still have these CLI commands.